src/deploy/RPM_build/noobaa.spec

```diff
@@ -48,23 +48,21 @@ ln -s /usr/local/noobaa-core/node/bin/node $RPM_BUILD_ROOT/usr/local/noobaa-core
 ln -s /usr/local/noobaa-core/node/bin/npm $RPM_BUILD_ROOT/usr/local/noobaa-core/bin/npm
 ln -s /usr/local/noobaa-core/node/bin/npx $RPM_BUILD_ROOT/usr/local/noobaa-core/bin/npx
 
-mkdir -p $RPM_BUILD_ROOT/etc/systemd/system/
-cp -R %{_builddir}/%{name}-%{version}-%{revision}/noobaa/src/deploy/noobaa_nsfs.service $RPM_BUILD_ROOT/etc/systemd/system/noobaa_nsfs.service
+mkdir -p $RPM_BUILD_ROOT/usr/lib/systemd/system/
+cp -R %{_builddir}/%{name}-%{version}-%{revision}/noobaa/src/deploy/noobaa_nsfs.service $RPM_BUILD_ROOT/usr/lib/systemd/system/noobaa_nsfs.service
 ln -s /usr/local/noobaa-core/src/deploy/nsfs_env.env $RPM_BUILD_ROOT/usr/local/noobaa-core/nsfs_env.env
 mkdir -p $RPM_BUILD_ROOT/etc/noobaa.conf.d/
 
 mkdir -p $RPM_BUILD_ROOT/etc/rsyslog.d/
-ln -s /usr/local/noobaa-core/src/deploy/standalone/noobaa_syslog.conf $RPM_BUILD_ROOT/etc/rsyslog.d/noobaa_syslog.conf
-ln -s /usr/local/noobaa-core/src/deploy/standalone/noobaa_rsyslog.conf $RPM_BUILD_ROOT/etc/rsyslog.d/noobaa_rsyslog.conf
+cp -R $RPM_BUILD_ROOT/usr/local/noobaa-core/src/deploy/standalone/noobaa_syslog.conf $RPM_BUILD_ROOT/etc/rsyslog.d/noobaa_syslog.conf
 
 mkdir -p $RPM_BUILD_ROOT/etc/logrotate.d/noobaa
 ln -s /usr/local/noobaa-core/src/deploy/standalone/logrotate_noobaa.conf $RPM_BUILD_ROOT/etc/logrotate.d/noobaa/logrotate_noobaa.conf
 
 %files
 /usr/local/noobaa-core
-/etc/systemd/system/noobaa_nsfs.service
+/usr/lib/systemd/system/noobaa_nsfs.service
 /etc/logrotate.d/noobaa/logrotate_noobaa.conf
-/etc/rsyslog.d/noobaa_rsyslog.conf
```

Comment:
@piste-jp-ibm now we dont have any reference to `noobaa_rsyslog.conf` in code, and we can delete this file.

Reply:
Removed from the tree.

Reply:
Sorry, I have realized `noobaa_rsyslog.conf` in the container environment. So I would like to keep `noobaa_rsyslog.conf` just in case.

My wish is removing that file by someone who have a responsibility of the container env. I believe my changes for `noobaa_rsyslog.conf` and `noobaa_syslog.conf` is equivalent to the previous code for the container environment.